Source/Scene/Instanced3DModel3DTileContent.js

```diff
@@ -394,6 +387,9 @@ define([
                     AttributeCompression.octDecodeInRange(octNormalUp[0], octNormalUp[1], 65535, instanceNormalUp);
                     AttributeCompression.octDecodeInRange(octNormalRight[0], octNormalRight[1], 65535, instanceNormalRight);
                     hasCustomOrientation = true;
+                } else if (hasTileTransform) {
+                    // Don't apply any orientation when the tile has a transform
+                    Matrix3.clone(Matrix3.IDENTITY, instanceRotation);
```

Comment:
For i3dm, a common use case is to position trees, telephone polls, etc. on the ground so using a default east-north-up is often right and saves space.  For interior use cases, like bolts, it is hard to define a useful default.

Perhaps the right approach is to add a boolean semantic to the feature table `EAST_NORTH_UP` rotation that defaults to `false`?  When `true`, it assumes the local `POSITION` (after quantization if used) is WGS84, and uses that for the local east-north-up frame?